homeassistant/components/esphome/climate.py

```diff
@@ -234,6 +240,16 @@ def current_temperature(self) -> float | None:
         """Return the current temperature."""
         return self._state.current_temperature
 
+    @property
+    @esphome_state_property
+    def current_humidity(self) -> int | None:
+        """Return the current humidity."""
+        return (
+            self._state.current_humidity
+            if self._static_info.supports_current_humidity
+            else None
+        )
```

Comment:
```suggestion
        if not self._static_info.supports_current_humidity:
            return None
        return self._state.current_humidity
```

Reply:
Ah yes they are way easier to understand =)